pkg/buffer/buffer_test.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/pion/rtcp"
 
+	log "github.com/pion/ion-sfu/pkg/logger"
```

Comment:
nit: Avoid renaming unless there's a collision.  I don't see one?

Reply:
Yes, just has been removed. As well as in sfu test .